src/integrated/soil_snow_model.jl

```diff
@@ -379,6 +379,16 @@ function soil_boundary_fluxes!(
         ) +
         p.excess_heat_flux +
         p.snow.snow_cover_fraction * p.ground_heat_flux
+
+    # Compute terms needed for derivatives
+    # ρc_sfc is stored in scratch! after we use it below, it may be overwritten
+    ρc_sfc = ClimaLand.Soil.get_ρc_sfc(Y, p, soil.parameters)
+    # Get the local geometry of the face space, then extract the top level
+    local_geometry_faceN =
+        ClimaLand.get_local_geometry_faceN(soil.domain.space.subsurface)
+    @. p.soil.dfluxBCdY.heat =

```

Comment:
are these multiplied by 0 just to exclude the contributions for now?

Reply:
yes! I wasnt sure if they would be initialized to zero, maybe so

Reply:
I guess also it was sort of a stand in for dF/dT